docs/Developers/Linter_Bears_Advanced.rst

```diff
@@ -41,6 +40,28 @@ The path of the temporary configuration file can be accessed inside
         def create_arguments(filename, file, config_file):
             return "--use-config", config_file
 
+If you return ``None``, no configuration file is generated. A common case
+where to explicitly return ``None`` is when the user wants to use his own
```

Comment:
maybe rephrase to
```
... is when you want to expose a setting for the user to use his own tool-specific config.
```
To clear the ambiguity between the user and the programmer, who wants what :D